repo/remote.go

```diff
@@ -117,7 +117,7 @@ type modVersionValue struct {
 // lookup logic should be moved to language/go. This means RemoteCache will
 // need to be initialized in a different way.
 type Repo struct {
-	Name, GoPrefix, Remote, VCS string
+	Name, GoPrefix, BuildNamingConvention, Remote, VCS string
```

Comment:
Commented above: I think we should either read this from `c.Repos` or build a `map[string]namingConvention` in `language/go.goLang.Configure` with this info. No need to add more Go-specific logic to this package.